scripts/build.py

```diff
@@ -263,14 +266,19 @@ def _compare_file_count(first_dir_path, second_dir_path):
     """Ensure that two dir's file counts match.
 
     Args:
-       first_dir_path: str. First directory to compare.
+       first_dir_path: str/list. First directory to compare.
```

Comment:
We use | not /, and also there is missing typeinfo for the list elements.

That said, this seems conceptually wrong. A dirpath is a string. It should not also be a list.

Reply:
Added new function.